src/query/graphite/native/aggregation_functions_test.go

```diff
@@ -22,6 +22,8 @@ package native
 
 import (
 	"fmt"
+	"github.com/golang/mock/gomock"
```

Comment:
nit: 3rd import group

Reply:
just so i don't make this mistake again:

WHY should this be  in the 3rd import group?